templates/zerver/app/left_sidebar.html

```diff
@@ -68,7 +68,7 @@
         </ul>
         <div id="streams_list" class="zoom-out">
             <div id="streams_header" class="zoom-in-hide"><h4 class="sidebar-title" data-toggle="tooltip" title="{{ _('Filter streams') }}">{{ _('STREAMS') }}</h4>
-                <i id="streams_inline_cog" class='fa fa-cog' aria-hidden="true" data-toggle="tooltip" title="{{ _('Subscribe, add, or configure streams') }}"></i>
```

Comment:
Is there any specific reason to remove `aria-hidden=true`?

Reply:
I have used jQuery to create a tooltip. If I don't remove `aria-hidden=true` then my new tooltip is not taken into consideration. My approach is similar to this PR #16571

Reply:
I think we will have to consider how this affects accessibility. https://github.com/zulip/zulip/commit/966bbf6b3d0d67027bdaf549baa80f65196d84d9 are the changes from that PR which were merged.